tests/e2e/tests/ChatComponent/ChatComponentVisible.ts

```diff
@@ -20,10 +20,13 @@ export default (
     await dashboard.navigateToDisasterType(dataset.hazard);
     // Assertions
     await userState.headerComponentIsVisible(dataset);
-    await chat.chatColumnIsVisibleForNoTriggerState({
-      user: dataset.user,
-      date,
-    });
+    if (dataset.scenario === 'no-trigger') {
```

Comment:
@gulfaraz I put in this check to account for the fact that this was a scenario-specific assertion. Are you ok with having this here, or should we only do this filtering on test-level in test.spec.ts (and thus skip this whole test for now if 'trigger').

Reply:
we want to work towards having such checks only at 1 place (either in test files or in test.spec.ts). Proabably the former. For now it's a mix.